### PR TITLE
update Macquarie Univ affiliations

### PR DESCRIPTION
**Pull request by lspitler**
A Department became a School and making other affiliations at Macquarie University consistent.

### CODE REVIEW

**Comment by pllim** [comment]
Can you please ping Jaime A. Alvarado-Montes and James Tocknell? Thanks!

**Comment by aragilar** [comment]
I'm happy with whatever Lee specifies, he's more familiar with the name changes than I am (@lspitler do you know if we still put the MQ in AAO-MQ, or has that now been dropped with the Astralis change, also do we need to add Astralis to our affiliations?).

**Comment by lspitler** [comment]
I think we're back to just AAO @aragilar , so maybe change it back if you can easily do so or I'll do so after I hear from Jaime. Astralis I guess not?

@JAAlvarado-Montes can you confirm you're happy with these changes?

**Comment by JAAlvarado-Montes** [comment]
Hi @lspitler @aragilar , I confirm everything looks good to me. Thanks!

Cheers,

Jaime A.

File: src/static/authors.tex [hunk 1]
```diff
@@ -87,9 +87,9 @@
 \affiliation{NSF's National Optical-Infrared Astronomy Research Laboratory, 950 North Cherry Avenue, Tucson, AZ 85719, United States}
 
 \author[0000-0001-6637-6922]{James~Tocknell}
-\affiliation{Department of Physics and Astronomy, Macquarie University, Sydney, NSW 2109, Australia}
+\affiliation{School of Mathematical and Physical Sciences, Macquarie University, Sydney, NSW 2109, Australia.}
```
Comment (pllim) [inline src/static/authors.tex:90]: Maybe remove the period to be consistent with other entries?
```suggestion
\affiliation{School of Mathematical and Physical Sciences, Macquarie University, Sydney, NSW 2109, Australia}
```

File: src/static/authors.tex [hunk 2]
```diff
@@ -118,8 +118,8 @@
 \affiliation{\affcfa}
 
 \author[0000-0003-0353-9741]{Jaime~A.~Alvarado-Montes}
-\affiliation{School of Mathematical and Physical Sciences, Macquarie University -- Sydney, NSW 2109, Australia.}
-\affiliation{Centre for Astronomy, Astrophysics and Astrophotonics, Macquarie University -- Sydney, NSW 2109, Australia.}
+\affiliation{School of Mathematical and Physical Sciences, Macquarie University, Sydney, NSW 2109, Australia.}
```
Comment (pllim) [inline src/static/authors.tex:121]:
```suggestion
\affiliation{School of Mathematical and Physical Sciences, Macquarie University, Sydney, NSW 2109, Australia}
```

File: src/static/authors.tex [hunk 3]
```diff
@@ -417,7 +417,9 @@
 \affiliation{\affipac}
 
 \author[0000-0001-5185-9876]{Lee~R.~Spitler}
-\affiliation{Department of Physics and Astronomy, Macquarie University, Sydney, NSW 2109, Australia}
+\affiliation{School of Mathematical and Physical Sciences, Macquarie University, Sydney, NSW 2109, Australia.}
```
Comment (pllim) [inline src/static/authors.tex:420]:
```suggestion
\affiliation{School of Mathematical and Physical Sciences, Macquarie University, Sydney, NSW 2109, Australia}
```

**Review by pllim — COMMENTED** [review]
Tsk tsk, no Oxford comma? 😉

**Review by pllim — APPROVED** [review]
Thanks!

**Pull request merged**